components/notifications-service/server/lib/formatters/slack.compliance.ex

```diff
@@ -26,8 +26,7 @@ defmodule Notifications.Formatters.Slack.Compliance do
 
     {profile_name, control_value} = collect_profile_and_control_names(failed_profiles, crit_failed_controls)
 
-    message = message_from_failed_controls(crit_failed_controls, compliance
-                                           .test_totals.critical)
```

Comment:
This was the newly found bug with this work. The `compliance.test_totals.critical` is the number of controls that failed not the number of tests.